ArcadiaParties.API/Startup.cs

```diff
@@ -23,6 +31,8 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
 
+            //temp SQLite connection
```

Comment:
Why "temp"? If you are going to change it later, why don't do this now? If not - why "temp"?

Reply:
Comment changed.  Watch new "Code style changes" PR